assets/scss/mq.scss

```diff
@@ -0,0 +1,8 @@
+@import '~sass-mq';
+
+$mq-responsive: true;
+
+$mq-breakpoints: (
+  desktop: 600px,
+  wide-desktop: 880px
```

Comment:
If 600px is "tablet", then 880px should be "desktop".

Reply:
Let's delay this for when we have desktop/mobile statistics se we can decide if adopting a "mobile-first"/"desktop-first" methodology.